scripts/development/m3_stack/start_m3.sh

```diff
@@ -243,6 +248,11 @@ echo "Validating namespace"
 [ "$(curl -sSf localhost:7201/api/v1/namespace | jq .registry.namespaces.metrics_30s_24h.indexOptions.enabled)" == true ]
 echo "Done validating namespace"
 
+echo "Waiting for namespaces to be ready"
+[ $(curl -sSf -X POST localhost:7201/api/v1/services/m3db/namespace/ready -d "{ \"name\": \"metrics_0_30m\", \"force\": true }" | grep -c true) -eq 1 ]
```

Comment:
Not for this PR - but probably worth adding an integration test once these PRs are landed that uses omits namespaces in the config and uses this instead.

Reply:
That's a great idea. Will definitely do that.